suggestions/rss_readers.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+        "id": "com.newsblur",
```

Comment:
linsui commented : _It's because the code is patched by me heavily. See https://gitlab.com/fdroid/fdroiddata/-/blob/master/metadata/com.newsblur.yml. Those patches may break in any future versions so the AF._

- Premium subscription option removed for F-Droid.
- Free account supports upto 64 sites. Others muted, but you can "Insta-fetch stories" for muted sites i.e. it'll update rss from muted sites if you update them manually. ([requested here](https://github.com/samuelclay/NewsBlur/issues/1635))
- Notable feature : Intelligence trainer.
- [Inoreader vs Newsblur](https://www.inoreader.com/alternative-to-newsblur)

Reply:
**This still breaks our requirements.** We require the project itself to be FOSS. F-Droid's meddling with the source code during the build-time is not justified in this case (or in any case for that matter). The only justification would be having a fork that removes those requirements.